src/analysis.py

```diff
@@ -0,0 +1,277 @@
+"""
+Author: Aidan Mattrick
+
+Date: Nov 28, 2020
+
+This script trains models and output results in the form of a figure to be used for further analysis.
+
+Usage: analysis.py -i=<input> -o=<output> [-v]
+
+Options:
+-i <input>, --input <input>     Local processed training data csv file in directory
+-o <output>, --output <output>  Local output directory for created pngs
+[-v]                            Report verbose output of dataset retrieval process
+"""
+
+import os
```

Comment:
We probably need to review which libraries we actually need for this next milestone and trim here.